src/client/search_requests.py

```diff
@@ -1177,7 +1108,7 @@ class AppBatchSearchRequest(BaseSearchRequest):
 
     Attributes:
         after_id: The ID of the batch used for cursor pagination.
-        status: Batches with the specified status.
+        status: Refer to the [AppBatch.Status](https://toloka.ai/en/docs/toloka-kit/reference/toloka.client.app.AppBatch.Status) page for more information on the available `status` values.
```

Comment:
Please replace this absolute link with a relative one (`toloka.client.app.AppBatch.Status.md`)